standalone/content/js/webapp.jsx

```diff
@@ -204,6 +204,9 @@ loop.webapp = (function(_, OT, mozL10n) {
     var participantStore = new loop.store.ParticipantStore(dispatcher);
     var serverConnectionStore = new loop.store.ServerConnectionStore(dispatcher, {});
 
+    dispatcher.dispatch(
+      new sharedActions.SetOwnDisplayName({ displayName: "Bill Smith" }));
```

Comment:
Eh. Maybe just "Bill Smith " + Math.random().toFixed(1).slice(-1) will be easier to read.